resources/event.py

```diff
@@ -30,23 +32,16 @@ def get(self):
 
     def post(self):
         # TODO: validate parameters
-        event = request.get_json(force=True)
-        creation_date = datetime.datetime.now()
-        event_date = event['eventDate']
-        title = event['title']
-        description = event['description']
-        court_id = event['courtID']
-        creator = event['creator']
-
-        # TODO: Add error handler
-
-        event = self.mongo.db.event.insert({
-            "creationDate": creation_date   ,
-            "eventDate": event_date,
-            "title": title,
-            "description": description,
-            "courtID": court_id,
-            "creator": creator
-        })
-
-        return eval(dumps(event)), 200
+        event_data = request.get_json(force=True, silent=True)
+
+        new_event = Event_model(
+            creation_date=datetime.datetime.now(),
+            event_date=event_data['eventDate'],
+            title=event_data['title'],
+            description=event_data['description'],
+            courtID=event_data['courtID'],
+            creatorUID=event_data['creator']
```

Comment:
this was changed in "events" branch. Did you merge it first?

Reply:
can't see any "events" branch